app/scripts/modules/titus/pipeline/stages/bake/titusBakeStage.js

```diff
@@ -77,7 +77,7 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.titus.titusBakeSt
 
       let buildParams = $.param(stage.repository.buildParameters);
       if (buildParams.length > 0) {
-        url += '&' + buildParams;
+        url += '&' + decodeURIComponent(buildParams);
```

Comment:
this will generate an invalid URL for things that actually need to be encoded, no?